tests/test_clients.py

```diff
@@ -1531,7 +1531,7 @@ def test_full_update_gebruiksrechten(self):
 
         # Create gebruiksrechten
         gebruiksrechten_data = {
-            "informatieobject": f"https://testserver/api/v1/documenten/{document.uuid}",
```

Comment:
Why was this changed? I prefer not to have specific URLs.

Reply:
These "https://testserver/..." urls fail when CI runs with turned on URL mapping, because this pattern is not known for set upt UrlMapping data. 
TNH I don't know why it worked before, but other Gebruiksrechten tests have "https://drc.utrechtproeftuin.nl/..." urls so I just adjusted these two tests to match

Reply:
Ok, then this will be moved to a new issue.